src/amitaRJun20/PrimeNumberCheck.java

```diff
@@ -0,0 +1,63 @@
+package amitaRJun20;
+
+public class PrimeNumberCheck {
+	// Method to check if a number is prime number or not
+	boolean primeNumber(int num) {
+		boolean flag = false;
+		for (int index = 2; index <= Math.sqrt(num); index++) {
+			if (num % index == 0) {
+				flag = true;
+				break;
+			}
+		}
+		return flag;
+	}
+
+	void isPrimeNumber(int num) {
+		if (primeNumber(num) == false)
+			System.out.println(num + " is a Prime number");
+		else
+			System.out.println(num + " is not a prime number");
+	}
+
+	// Method to find count of all prime number between 2 & 100
+	int countPrimeNumbers(int startnum, int endnum) {
+		int count = 0;
+		for (int index = startnum; index <= endnum; index++) {
+			if (primeNumber(index) == false) {
+				count++;
+			}
+		}
+		return count;
+	}
+
+	//// Method to find sum of all prime number between 2 & 100
+	int sumAllPrimeNumbers(int startnum, int endnum) {
+		int sum = 0;
+		for (int index = startnum; index <= endnum; index++) {
+			if (primeNumber(index) == false) {
+				sum += index;
+			}
+		}
+		return sum;
+	}
+
+	// Method to find average of all prime number between 2 & 100
+	double averageOfAllPrimeNumbers(int startnum, int endnum) {
```

Comment:
No need to write extra methods, Just create two instance variable sum and count and perform that operation in isPrime method and use those variable in method like:
void findPrimeNumberInGivenRange(int startNumber, int endNumber) {
for(int index=startNumber; index<=endNumber; index++) {
primeNumber(index);
}
System.out.println("Total count is: "+count);
System.out.println("Sum of all Prime number is: "+sum);
System.out.println("Average of all prime number is: "+sum/count);